docker/gatling/notify.sh

```diff
@@ -4,16 +4,14 @@ set -o errexit
 set -o nounset
 set -o verbose
 
-TOPIC_NAME=$1
+TOPIC_ARN=$1
```

Comment:
This does diverge this file from the `notify.sh` but I think it's ok so far (if we reuse this again then let's rethink).